addons/main/functions/fnc_regenBleedout.sqf

```diff
@@ -0,0 +1,40 @@
+/*
+    Credit to MajorDanvers
+    Slowly regenerates bleedout timer, faster when closer to death
+    2 minutes to regen last down to second last
+    15 minutes to regen first down
+    For coeff == 1, 1 second regenerated per duration of bleedout timer
+*/
+#include "script_component.hpp"
+params ["_unit"];
+
+#define malus GVAR(bleedOutTimeMalus)
+#define bleedout GVAR(bleedoutTime)
+#define subtraction GVAR(bleedoutTimeSubtraction)
+#define coeff GVAR(bleedoutRegenCoeff)
```

Comment:
Agreed. I was getting ready to remove them, since `malus` is the only one saving space anyway.
I hadn't noticed those `{}` in the if statements. Are they still necessary, or just an artifact from the original script?

Reply:
The braces in if statements are to force lazy evaluation since Arma doesn't do it by default, `isNil { }` vs `isNil "variable"` is what diwako means.